### PR TITLE
Fixed UI display issue in the installation workflow (bsc#1117492)

### PR DESCRIPTION
**Pull request by lslezak**
- https://bugzilla.suse.com/show_bug.cgi?id=1117492
- Fixes a not cleared screen
- 4.1.10

## After Applying the Fix

registration

### CODE REVIEW

File: src/lib/registration/clients/inst_scc.rb [hunk 1]
```diff
@@ -320,6 +322,8 @@ def start_workflow
     def workflow_start
       log.debug "WFM.Args: #{WFM.Args}"
 
+      Yast::Wizard.ClearContents
```
Comment (dgdavid) [inline src/lib/registration/clients/inst_scc.rb]: Just a question: Why is this needed here if now the BaseSystemRegistration dialog is ensuring to clear the content?

Comment (lslezak) [reply src/lib/registration/clients/inst_scc.rb]: Ah, right, this was my initial test, I forgot to remove it, it works without it.
